.gitpod.yml

```diff
@@ -0,0 +1,3 @@
+tasks:
+  - init: git submodule init && git submodule update
+    command: git submodule update
```

Comment:
https://www.gitpod.io/docs/config-start-tasks/#defining-commands

Reply:
hm, why would you always want to call `git submodule update` every time? I see that the init command is necessary but the submodule won't change unless a new version is checked out?

Reply:
I followed https://www.gitpod.io/docs/cpp-in-gitpod/

which suggests

https://github.com/Circuito-io/ComponentEditor/blob/master/.gitpod.yml

I don't see why this hurts, because this will only be executed if you startup the workspace.

Reply:
alright